src/cider/tests/nextgen/jitlib/JITLibTest.cpp

```diff
@@ -540,6 +540,41 @@ TEST_F(JITLibTests, NestedForControlFlowTest) {
   EXPECT_EQ(func_ptr(), 100000);
 }
 
+TEST_F(JITLibTests, ForContinueTest) {
+  LLVMJITModule module("TestModule");
+  JITFunctionPointer func =
+      JITFunctionBuilder()
+          .setFuncName("test_func")
+          .registerModule(module)
+          .addReturn(JITTypeTag::INT32)
+          .addProcedureBuilder([](JITFunctionPointer function) {
+            JITValuePointer ret = function->createVariable(JITTypeTag::INT32, "ret", 0);
+
+            auto loop_builder = function->createLoopBuilder();
+            JITValuePointer index =
+                function->createVariable(JITTypeTag::INT32, "index", 10);
+
+            loop_builder->condition([&]() { return index + 0; })
+                ->loop([&](LoopBuilder*) {
```

Comment:
seems you capture the loop_builder. then why do you need LoopBuilder* arguments?